enterprise/api/v1/views.py

```diff
@@ -273,6 +275,15 @@ class LicensedEnterpriseCourseEnrollmentViewSet(EnterpriseWrapperApiViewSet):
     queryset = models.LicensedEnterpriseCourseEnrollment.objects.all()
     serializer_class = serializers.LicensedEnterpriseCourseEnrollmentReadOnlySerializer
 
+    class RevocationStatus:
```

Comment:
Should this be in here? Would it make sense for it to have its own file/be in a constants file?

Reply:
This is a set of constants that's only relevant to this ViewSet, with a namespace.  I don't think it makes sense to extract it, given that no other ViewSet should use it.